### ISSUE TITLE
No Virtual camera button on linux kernel 5.18

### ISSUE THREAD

**Issue by hubyhuby** [obsproject/obs-studio#7124]
### Operating System Info

Ubuntu 22.04

### Other OS

Kernel 5.18 instead of 5.15 LTS

### OBS Studio Version

27.2.4

### OBS Studio Log URL

https://obsproject.com/logs/yC2CzGdLvd0D0czO

### OBS Studio Crash Log URL

https://obsproject.com/logs/yC2CzGdLvd0D0czO

### Expected Behavior

The Virtual Camera button should be present.

### Current Behavior

The video is showing and everything seems to work.
BUT the button "start virtual camera is not there".

WIth kernel 5.15
![Scr_33_3](https://user-images.githubusercontent.com/26340942/185366136-37ae6420-a2dc-4b1d-8aa4-348f53494874.png)

### Steps to Reproduce

1.Install ubuntu 22.04 (mate)
2.Switsh to the 5.18 kernel (instead of the default 5.15)
3.Launch OBS
4 see that there is no button
5 switsh back to 5.15 kernel => the button is back

With kernel 5.19 : no button :
![Scr_33_2](https://user-images.githubusercontent.com/26340942/185366209-128ff9bf-726c-4d75-be9f-0cb1e8340dd1.png)

### Anything else we should know?

This is anoying because on my upcoming intel 12th gen , I need to switsh to kernel 5.18.
For now using intel i3 8145U

**Comment by WizardCM** [comment]
This sounds more likely to be a support issue than a bug. Please submit this through one of our support channels below, and if it's determined that this is indeed a bug, then we will re-open this issue.

Please use the [Discord server](https://obsproject.com/discord) or the [forums](https://obsproject.com/forum). For more info, check our [Help page](https://obsproject.com/help).

**Comment by hubyhuby** [comment]
Dear @WizardCM ,
I doesn t look like a support issue.
Since it does work verywell with linux kernel 5.15.
I just have to reboot to 5.15 "et voila".
I have not change any configuration...
I just rebooted a second time select kernel 5.18 and bam no button again.
![image](https://user-images.githubusercontent.com/26340942/185368158-5c6f6c87-5ee4-4dbd-be22-5bb9edbe0ecc.png)

Don t know what else to do on the forum.
Feel free to reopen. 
Have a nice day.

**Comment by WizardCM** [comment]
Yes, the virtual camera (`v4l2loopback dkms`) is a kernel module and has to be rebuilt for your new kernel.

**Comment by hubyhuby** [comment]
Thank you for your help.
I ll just stick with 5.15 kernel for OBS then .
Thsk for this super software ! <3

**Comment by joncline** [comment]
How is this a support issue? This will not work on the latest upgrade to Ubuntu using their Software tool:

Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy

This is a major issue for remote workers using OBS virtual backgrounds.

**Comment by joncline** [comment]
I tried building OBS using these directions though the Virtual Camera button does not appear: https://obsproject.com/wiki/build-instructions-for-linux

Anyone know where directions are to build it with a Virtual Camera for this version of Ubuntu? Ubuntu 22.04.1 LTS

**Comment by hubyhuby** [comment]
@strk Very interesting that there is a fix from the kernel on its way to our distro.
Have you tried the fix ? Do you have an easy work around like installing a ".deb" . Couldn t find the deb for "ubuntu6". I guess I will have just to wait for the update on my distro.
 I found 0.12.7 but it seems a little risky to install for now :
	v4l2loopback-dkms_0.12.7-2ubuntu1_all.deb	2022-08-16 15:44	29K
![image](https://user-images.githubusercontent.com/26340942/189472619-08cd978d-1a8c-469f-b0b6-9e9829be8668.png)


[v4l2loopback-dkms_0.12.7-2ubuntu1_all.deb](http://us.archive.ubuntu.com/ubuntu/pool/universe/v/v4l2loopback/v4l2loopback-dkms_0.12.7-2ubuntu1_all.deb)	2022-08-16 15:44	29K

**Comment by strk** [comment]
>  Have you tried the fix ?

Yes, worked for me, I did not install a new deb, just the one shipped with my system (v4l2loopback-dkms),
which install a source tree under /usr/src/v4l2loopback-0.12.5/

I've then patched the /usr/src/v4l2loopback-0.12.5/dkms.conf file as instructed and then:
```
sudo dkms build -m v4l2loopback/0.12.5
sudo dkms autoinstall v4l2loopback
sudo modprobe v4l2loopback card_label="OBSCam" exclusive_caps=1
```
